### DISCUSSION TITLE
Can analog stick act as ABS (absolute coordinates)?

### DISCUSSION THREAD

**Discussion by Trihy** [flyinghead/flycast#1405] [Ideas]
I wonder if will be possible to add analog stick as Absolute coordinates instead of relative.

It will be handy for DC multiplayer lightgun games and lightguns configuration. Since many of them support analog stcik mapping but need to be absolute coordinates.

It could be just an option on standalone or core to use analog stick as ABS or REL.

**Comment by flyinghead** [comment]
This is already supported for arcade games.
All lightgun Dreamcast games also support gamepads so this isn't needed.

**Reply by Trihy** [reply]
Sure. The problem will be DC games with lightguns. Since lightguns need absolute coordinates. DC uses for their games relative input. You can change input to lightgun or mouse, but that will be a problem for multiplayer games.

I saw some users asked for relative input support on arcade games. This will be the opposite for DC games.
